src/commands/init.ts

```diff
@@ -187,19 +193,55 @@ export default class Init extends Command {
     }
     packageJson.scripts = { ...packageJson.scripts, ...scripts };
 
-    if (initializationType === InitType.freshExpress) {
-      packageJson.devDependencies = { ...packageJson.devDependencies, ...packageJsonData.devDependencies };
-    }
+    packageJson.dependencies = await this.addDependencies(packageJsonData.dependencies, packageJson.dependencies, flags.projectDir);
```

Comment:
``` ts
packageJson.dependencies = { ...packageJson.dependencies, ...(await this.addDependencies(packageJsonData.dependencies)) }
```

Reply:
Also might be worth it now to change it to 
```ts
packageJson = { 
  ...packageJson, 
  scripts: { ...packageJson.scripts, ...scripts },
  dependencies: { ...packageJson.dependencies, ...(await this.addDependencies(/* ... */)) },
  devDependencies: { ...packageJson.devDependencies, ...(await this.addDependencies(/* ... */)) }
}
```

and maybe move out the script part from line 173 to 187 to a separate function as well.

Reply:
I have to get used to the spread operator. It can really do a lot for you.